pkg/liqoctl/uninstall/handler.go

```diff
@@ -66,7 +66,10 @@ func (o *Options) Run(ctx context.Context) error {
 
 	s := o.Printer.StartSpinner("Running pre-uninstall checks")
 	if err := utils.PreUninstall(ctx, o.CRClient); err != nil {
-		s.Fail("Pre-uninstall checks failed: ", output.PrettyErr(err))
+		errMsg := fmt.Sprintf("Pre-uninstall checks failed: %s\n"+
+			"You can disable the active peerings with other clusters with the 'liqoctl unpeer' command.\n"+
```

Comment:
I would say "You must tear down the active peerings..." instead of "You can disable the active peering...."
Saying "you can", it means (to me) that it's just a suggestion, not a mandatory requirement to proceed.

Reply:
The issue here is that the errors might not be related to the active peerings, but as @aleoli said, there might be also the possibility that there still is an offloaded namespace or other resources to remove. So I wouldn't always print to "You must tear down the active peerings..." because this is not always the case.
We might add at the end of the error message something like:
- To disable active peering launch liqoctl X
- To unoffload a namespace launch liqoctl Y

Actually, the ideal solution would be changing the returned error message to suggest a solution for each of the returned error, but since this is a shared function, I wouldn't suggest to use `liqoctl` when the user is not actually using it. So we should add a logic to understand when we are using liqoctl and print the suggestion. However, I'm not sure if it is worth it.
What do you guys think? @aleoli @fra98 @cheina97 

Maybe in future, it would be nice having a flag like `--force` that automatically removes everything that prevents Liqo from being uninstalled.

Reply:
Is there any command we can suggest to look for active resources? (e.g., some "status"?)
In this case, the message could be "cannot uninstall because there are still some active resources. Check the status of your setup with command XXX and clean them properly to uninstall, or use the "--force" flag.

Reply:
We finally decided to proceed with the best solution for the user: for each of the problems that prevent Liqo from being uninstalled we suggest the resolution